CHANGELOG.md

```diff
@@ -6,6 +6,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.2.0]
+### Added
+- Add `init` command ([#77](https://github.com/MetaMask/auto-changelog/pull/77))
+
+### Changed
+- Validate changelog during CI ([#79](https://github.com/MetaMask/auto-changelog/pull/79))
+- Cache npm dependencies ([#80](https://github.com/MetaMask/auto-changelog/pull/80))
+- Add `@lavamoat/allow-scripts` and `setup` command ([#78](https://github.com/MetaMask/auto-changelog/pull/78))
+- Use `actions/setup-node@v2` instead of `actions/setup-node@v1` ([#81](https://github.com/MetaMask/auto-changelog/pull/81))
+
+### Fixed
+- Fix changelog where first release isn't first ([#76](https://github.com/MetaMask/auto-changelog/pull/76))
```

Comment:
This message could be improved.

Reply:
:joy: I swear I just read this and thought "OK, this is fine, this makes sense.". It does not make sense.

Reply:
Hopefully it makes more sense now: https://github.com/MetaMask/auto-changelog/pull/82/commits/d432abe72d50bc07f6a71b18916f0179890b6449

Reply:
lol, that's better! 😂